src/modules/launcher/PowerLauncher/Helper/ThemeExtensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using ManagedCommon;
+using Microsoft.Win32;
+using Wpf.Ui.Appearance;
+
+namespace PowerLauncher.Helper
+{
+    public static class ThemeExtensions
+    {
+        public static Theme ToTheme(this ApplicationTheme applicationTheme)
```

Comment:
Done in order to not break plugins API: we are still using `Theme` enum from `ManagedCommon`. Does it make sense to create an enum dedicated to PT Run?